nuxeo-connect-client/src/main/java/org/nuxeo/connect/data/PackageDescriptor.java

```diff
@@ -240,21 +262,21 @@ public String getConflictsAsString() {
         return optionalDependencies;
     }
 
-    @JSONExportMethod(name = "dependencies")
-    protected JSONArray getDependenciesAsJSON() {
+    @JSONExportMethod(name = "optionalDependencies")
+    protected JSONArray getOptionalDependenciesAsJSON() {
         JSONArray deps = new JSONArray();
-        for (PackageDependency dep : getDependencies()) {
+        for (PackageDependency dep : getOptionalDependencies()) {
             deps.put(dep.toString());
         }
         return deps;
     }
 
-    public String getDependenciesAsString() {
+    public String getOptionalDependenciesAsString() {
         StringBuilder sb = new StringBuilder();
```

Comment:
ok for @since but the initial method is still here, I think the diff is strange because I moved the getDependenciesAsString and getDependenciesAsJSON methods (to order them properly in the file) and added getOptionalDependenciesAsString and getOptionalDependenciesAsJSON